SerialPrograms/Source/PokemonSV/Inference/ItemPrinter/PokemonSV_ItemPrinterPrizeReader.cpp

```diff
@@ -117,7 +124,75 @@ std::array<std::string, 10> ItemPrinterPrizeReader::read(
 }
 
 
+std::array<int16_t, 10> ItemPrinterPrizeReader::read_quantity(

```

Comment:
Given how big and invasive this PR is, let's break it up. Can start by splitting out this quantity reader.